app/components/layout/TopBar.tsx

```diff
@@ -119,15 +120,17 @@ export default function TopBar({ button, onHamburgerClick }: TopBarProps) {
       </StyledLogoWrap>
 
       <SelectWrapper>
-        <Select
-          options={languageOptions}
-          selectedValue={selectedLanguage}
-          setSelectedValue={(language) => {
-            setSelectedLanguage(language);
-            ga("global language select", "language change", language.value);
-          }}
-          autoWidth
-        />
+        <FormControl sx={{ minWidth: 200 }} size="small">
+          <SelectMui
+            label="Select Language"
```

Comment:
I used "Select Language" instead of "View as" from Figma, but I'm happy to change it back to that or something else, if we don't like this.

Reply:
Select Language fits best here.